### ISSUE TITLE
Invalid authentication for compute tring to connect my GNS3 VM

### ISSUE THREAD

**Issue by CyanSlayer** [GNS3/gns3-server#2462]
Hi so i got this error since i got the latest version GNS3 3.0.0, and since there is no GNS3 VM on the setting for my Vmware GNS3  i am trying to connect my GNS3 VM on my Vmware via Remote Compute  i have enter the default password but i got this invalid authentication  but when i connect it as a main remote controller and enter the exact same default username, password and port number it works.  
![remotecontrolworks](https://github.com/user-attachments/assets/e06b8bc0-3094-40a2-a982-979b6ba4e243)
![remotecomputedoesnotwork](https://github.com/user-attachments/assets/78d14339-d37b-4ff7-bb8e-3406e26550db)

**Comment by grossmj** [comment]
You can only add a controller in the Setup Wizard or Controller preferences. Computes must be added in the remote computes section:

![Image](https://github.com/user-attachments/assets/154af6e7-9432-4bd7-a57a-cbe40c0156d7)

**Comment by CyanSlayer** [comment]
oh so the Remote computes is only for the local host which is my main operating system and i can only add the single controller on either the wizard or Controller setting.

**Comment by grossmj** [comment]
Yes, there should only be one controller, then you can add one or more computes to spread the load on multiple hosts/machines.

**Comment by blair19** [comment]
When I attempt to add Remote Computes in v3.0.3 on Ubuntu 24.04.2, I receive the following error. Both servers are running same version, credentials entered for the remote compute are correct as they work for its web GUI. I've attempted creating different users along with toggling settings in the thick client in attempts to get this to work, but I'm coming up empty for both web and client.

ERROR server.py:144 Controller unauthorized error in /v3/computes/74bb350c-50f0-4f42-b7d4-1368131dd809/connect (POST): Invalid authentication for compute 'Compute1' [74bb350c-50f0-4f42-b7d4-1368131dd809]

**Comment by markooff** [comment]
Today is 09 .01. 2026 - and THIS PROBLEM - STILL EXISTS ! 
(and preventing people for using external GNS3_VM at all .... ) 

I did some tests in my environment (I upgraded it from 2.2.46 version - both GNS3_server and GNS3_VM  where everything was working correctly (!) ). First I created completely new pair of credentials in config file in GNS3_VM : 

[config on GNS3_VM](https://markooff.net/obrazki/GitHub/GNS3/error_3_0_5_with_authorization/2026-01-09_02_20_31-root@gns3vm__~.png)

But when I added this machine (GNS3_VM) into my configuration in GNS3_main_controller - as shown on the next screen : 

[config: GNS2_VM in main cotroller's config](https://markooff.net/obrazki/GitHub/GNS3/error_3_0_5_with_authorization/2026-01-09_02_23_28-debianGNS3_2_(markooff)_-_TigerVNC.png)

and tried to connect them both (GNS mail controller with GNS3 VM as remote server ) 
I always got : 

[error: ](https://markooff.net/obrazki/GitHub/GNS3/error_3_0_5_with_authorization/2026-01-09_02_24_28-debianGNS3_2_(markooff)_-_TigerVNC.png)

Moreover, I did a little research - on what's on the wire :) - using tcpdump on GNS3_VM machine , and Wireshark as pcap viewer/editor - and it showed me that during all connection attempts  - there were GOOD (Proper!) pair [username and password] sent to the GNS3_VM (!!) , but PROBLEM was only with proper understanding and procesing this credentials on the GNS3_VM (3.0.5) authorization mechanism ...

[http-request for authorization - packet 42 ](https://markooff.net/obrazki/GitHub/GNS3/error_3_0_5_with_authorization/2026-01-09_12_16_20-zzut_polacznie_3_0_5__001.png)

[http-json answer with error procesing credentials  - packet 46](https://markooff.net/obrazki/GitHub/GNS3/error_3_0_5_with_authorization/2026-01-09_12_18_51-zzut_polacznie_3_0_5__001.pcap.png)


So there's no problem - with sending unproper login/pass pair on the wire !
But there's still problem with answearing on the lgin attempts (!!!!!!!!) 

Could someone from your team (GNS0-server) seriously address this problem? :) 
And solve it eventually .... ?
